plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -658,10 +659,9 @@ private void addSplitPluginDependencies(String thisPlugin, MavenRunner.Config mc
                     System.out.println("Skipping implicit dep " + thisPlugin + " → " + plugin);
                     continue;
                 }
-                VersionNumber splitPoint = new VersionNumber(pieces[1]);
+                String splitPoint = pieces[1];
                 VersionNumber declaredMinimum = new VersionNumber(pieces[2]);
-                // TODO this should only happen if the tested core version is ≥ splitPoint
-                if (coreDep.compareTo(splitPoint) <= 0 && !pluginDeps.containsKey(plugin)) {
+                if (coreCoordinates.compareVersionTo(splitPoint) >= 0 && !pluginDeps.containsKey(plugin)) {
```

Comment:
Being the second comparison '<=' but yes.

I am not sure of following you here Kristin. Could you explain further? (maybe with an example?). My thinking was:
- One plugin was split out of the core in a certain version.
- If the baseline for the plugin is smaller, then the plugin will not include the dependency to such plugin.
- If PCT is executed against a greater core version, then we need to include the dependency.
- However if the plugin's baseline is bigger than the split version, it will already include the dependency so we do not need to include it even if executing with a greater version.

Am I wrong?